src/nonparametricbootstrap.jl

```diff
@@ -102,7 +93,7 @@ function nonparametricbootstrap(rng::AbstractRNG, n::Integer,
 end
 
 resample!(mod::LinearMixedModel, blups=ranef(mod), reterms=mod.reterms) =
-    resample!(GLOBAL_RNG, mod, blups, reterms)
+    resample!(Random.GLOBAL_RNG, mod, blups, reterms)
```

Comment:
ah nice, I stumbled over this one but didnt know if it is a bug or not :S next time I will report early promise